array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,22 @@
+/* exported take */
+/*
+  set output equal to empty array
+  set i equal to zero
+  if array count is greater than array length
+    return empty array
+  while i is less than count,
+    append array at i to output
+    increment i by  one
+  return output
+
+*/
+function take(array,count){
+  var output = [];
+  if (count > array.length) {
+    return [];
```

Comment:
In this case, if the count is more than the size of the array, the entire array should be given back since that would mean you are "taking" all the data inside of it.